lingua_franca/format.py

```diff
@@ -578,3 +578,53 @@ def nice_response(text, lang=''):
         assertEqual(nice_response_de("10 ^ 2"),
                          "10 hoch 2")
     """
+
+
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def nice_bytes(number, lang='', speech=True, binary=True, gnu=False):
+    """
+    turns a number of bytes into a string using appropriate units
+    prefixes - https://en.wikipedia.org/wiki/Binary_prefix
+    spoken binary units - https://en.wikipedia.org/wiki/Kibibyte
+    implementation - http://stackoverflow.com/a/1094933/2444609
+    :param number: number of bytes (int)
+    :param lang: lang_code, ignored for now (str)
+    :param speech: spoken form (True) or short units (False)
+    :param binary: 1 kilobyte = 1024 bytes (True) or 1 kilobyte = 1000 bytes (False)
+    :param gnu: say only order of magnitude (bool)  - 100 Kilo (True) or 100 Kilobytes (False)
```

Comment:
Maybe note dependency on `binary` flag here; `kilobytes`/`kibibytes` since if True, it's always the base-10 prefix but false is dependent on `binary`